### PR TITLE
fix: Changed From method to get the AVD name from the config.ini

### PR DESCRIPTION
**Pull request by jguadagno**
fix :Changed From method to get the AVD name from the config.ini

Reorganized the `From` method to extract the name of the AVD from the `config.ini` so that the *<avd_name>.ini* could be built. 

Refs: #20

### CODE REVIEW

**Comment by kazo0** [comment]
Thanks for the PR! I have allowed the CI to run its checks but I will leave the approval and merge to maybe @davidjohnoliver since I don't know too much about the internals of uno-check

**Comment by agneszitte-nventive** [comment]
Hi @jguadagno, thank you for the PR!

Just to let you know that all commits must be in the Conventional Commits format, please.
We use this to automatically generate release notes for new releases.

The expected format for the commit is for example:
fix: Changed From method to get the AVD name from the config.ini
(For more details, see https://www.conventionalcommits.org for help on how to structure commits)

**Comment by jguadagno** [comment]
@agneszitte-nventive Sorry, I did not know that the Conventional Commit format was required for commits to this repo also.  I've edited the title and description. I hope it fits the conventions.

**Comment by Xiaoy312** [comment]
@jguadagno You need to change the commit message itself, not the pr title.

**Comment by jguadagno** [comment]
@Xiaoy312 Hopefully I got it right now :(  I'm a bit of a git newbie

File: UnoCheck/AndroidSdk/Avd.cs [hunk 1]
```diff
@@ -60,6 +68,9 @@ public static Avd From(string avdPath)
 							return avd;
 						}
 					}
+
+
+
```
Comment (Xiaoy312) [inline UnoCheck/AndroidSdk/Avd.cs:73]: please edit those lines out

Comment (jguadagno) [reply UnoCheck/AndroidSdk/Avd.cs]: Done

File: UnoCheck/AndroidSdk/Avd.cs [hunk 2]
```diff
@@ -36,20 +36,28 @@ public static Avd From(string avdPath)
 			{
 				try
 				{
-					var avdIniFile = System.IO.Path.Combine(avdPath, "..", System.IO.Path.GetFileNameWithoutExtension(avdPath) + ".ini");
+
```
Comment (Xiaoy312) [inline UnoCheck/AndroidSdk/Avd.cs:39]: empty line

File: UnoCheck/AndroidSdk/Avd.cs [hunk 3]
```diff
@@ -59,7 +67,7 @@ public static Avd From(string avdPath)
 
 							return avd;
 						}
-					}
+					}
```
Comment (Xiaoy312) [inline UnoCheck/AndroidSdk/Avd.cs:70]: trailing spaces

**Review by Xiaoy312 — CHANGES_REQUESTED** [review]
@jguadagno you are supposed to put the changes into one commit with the appropriate names, not stacking more commit on top. 
fyi, you can use `git reset --soft` to tear down a commit and then use `git commit --amend` to fuse the changes into previous commit. alternative `git rebase -i HEAD~4`, fine too.
since you are unfamiliar with git, i would advice you to copy the content of current file somewhere, before proceeding as it is possible to lose the work if you do mess up.
also this may help: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
note that, since you dont have much changes, another option to start over and just paste in what you have saved if you are having trouble with git.

**Comment by jguadagno** [comment]
Created a new PR #23

**Pull request closed without merging**